### PR TITLE
Particle ID

### PR DESCRIPTION
**Pull request by Duchstf**
initial codes for particle ID algo

### CODE REVIEW

File: packages/reco/ktracker/SQReco.cxx [hunk 1]
```diff
@@ -436,12 +439,31 @@ int SQReco::process_event(PHCompositeNode* topNode)
     if(is_eval_dst_enabled()) _tracklet_vector->push_back(&(*iter));
     ++nTracklets;
   }
-
+  // Pick up conversation
```
Comment (wpmccormack) [inline packages/reco/ktracker/SQReco.cxx]: Would you mind deleting this line?

File: packages/reco/ktracker/SQReco.cxx [hunk 2]
```diff
@@ -436,12 +439,31 @@ int SQReco::process_event(PHCompositeNode* topNode)
...
   std::sort (temporarySTracks.begin(), temporarySTracks.end(), compRecTrack);
   for(unsigned int st = 0; st<temporarySTracks.size(); st++){
     fillRecTrack(temporarySTracks.at(st));
   }
 
+  //>>>>>>>>>>>>>>>>>PARITCLE ID
+  std::cout << "Here 1" << std::endl;
+  ParticleID();
+  std::cout << "Here 2" << std::endl;
+  //<<<<<<<<<<<<<<<<<<<<<<<<<<<<
```
Comment (wpmccormack) [inline packages/reco/ktracker/SQReco.cxx]: could you comment out the debugging lines?

File: packages/reco/ktracker/SQReco.cxx (hunk 1 above)
Comment (Duchstf) [reply packages/reco/ktracker/SQReco.cxx]: AH ok sorry!

File: packages/reco/ktracker/SQReco.cxx (hunk 2 above)
Comment (Duchstf) [reply packages/reco/ktracker/SQReco.cxx]: 👍

**Pull request merged**